independent-publisher-connectors/ORB Intelligence/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4716 @@
+                                                        "example": "1231"
+                                                    },
+                                                    "cidrs_count": {
+                                                        "type": "integer",
+                                                        "format": "int32",
+                                                        "example": 18
+                                                    },
+                                                    "liveramp_idl_count": {
+                                                        "type": "integer",
+                                                        "format": "int32",
+                                                        "example": 7
+                                                    },
+                                                    "liveramp_device_count": {
+                                                        "type": "integer",
+                                                        "format": "int32",
+                                                        "example": 432
+                                                    },
+                                                    "latitude": {
+                                                        "type": "string"
+                                                    },
+                                                    "longitude": {
+                                                        "type": "string"
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        "required": [
+                                            "country",
+                                            "iso_country_code",
+                                            "name",
+                                            "orb_num"
+                                        ]
+                                    }
+                                }
+                            }
+                        },
+                        "description": "List of found companies."
+                    }
+                },
+                "tags": [
+                    "Search and Look-alike"
+                ],
+                "summary": "Retrieves a list of companies similar to a target company",
+                "description": "The Lookalike API call allows a user to retrieve a list of companies similar to a target company and can be narrowed down based on a range of filtering attributes. This enables users to expand their total addressable market or evaluate leads by identifying for companies that look like existing customers. The input to the Lookalike call are the Orb Number of the target company (mandatory field) and a list of filt
```

Comment:
@aaryan2134 The above description seems invalid or has over information than needed, please look into it and make it shorter.

Reply:
Done